core/questions.csv

```diff
@@ -4,6 +4,10 @@ How many runners should the scheduler manage?,SCHEDULER,num_runners,4
 How often should the scheduler run in minutes?,SCHEDULER,frequency_in_mins,1
 Where is the weights directory?,RUNNER,weights_directory,weights
 What is the max number of times a job should run?,SCHEDULER,max_tries,3
+What is the port for IPFS?,BLOCKCHAIN,ipfs_port,5001
+What is the port for HTTP?,BLOCKCHAIN,http_port,3000
+What is the host for HTTP?,BLOCKCHAIN,host,127.0.0.1
+What is the timeout for HTTP?,BLOCKCHAIN,timeout,5
```

Comment:
Are these questions that data provider will actually answer? If not, please change the question entry in all of them to `SECRET`.

Reply:
The first three will be answered, I think. Last changed to SECRET. Easy enough to change in future.

Reply:
Sounds good. Not 100% sure whether Config Manager supports sections whose questions are only partially secret, so just a heads up (trivially you could just change the name of the section for the last question to `BLOCKCHAIN_SECRET` if it doesn't work.